pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -132,7 +132,7 @@ func readNextMessages(
 }
 
 func stripTsFromPayloads(
```

Comment:
i dont think this is the right place for this / the right factoring. you're stripping source now too, and also asserting things about it